.lando.yml

```diff
@@ -22,22 +23,22 @@ services:
         SSH_AUTH_SOCK: "/run/host-services/ssh-auth.sock"
         GEM_HOME: "/usr/local/bundle"
         PATH: "/usr/local/bundle/bin:/usr/local/bundle/gems/bin:${PATH}"
+        QT_QPA_PLATFORM: offscreen
       ports:
         - '18880:18880'
     command:
       - if [ ! -L /var/www/.aws ]; then ln -s /user/.aws /var/www/.aws; fi
       - /app/gulp
     build_as_root:
-      - apt-get update -qq
-      - apt-get install -y apt-transport-https
-      - apt-get install -y vim net-tools
+      - apt-get update -qq && apt-get install -y apt-transport-https && apt-get install glibc-source
+      - apt-get install -y phantomjs vim net-tools
       - curl -sL https://deb.nodesource.com/setup_16.x | bash -
       - curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
       - echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
       - apt-get update -qq
       - apt-get autoclean
       - apt-get install mariadb-client build-essential libpq-dev yarn nodejs -y
-      - gem install bundler
+      - gem install bundler && bundle config set force_ruby_platform true
```

Comment:
this may not be necessary, and may be making everything slower... try without it